package.json

```diff
@@ -90,6 +90,7 @@
   "packageManager": "yarn@3.2.2",
   "dependencies": {
     "@exodus/migrate": "^1.5.3",
+    "delay": "^5.0.0",
```

Comment:
tmp, will update atoms in a separate PR https://github.com/ExodusMovement/exodus-hydra/pull/5307

Reply:
updated in #16